client/app/components/Tooltip/Tooltip.scss

```diff
@@ -12,16 +12,16 @@
   }
 
   > div:last-of-type {
-    @include setFontSize($size-18);
-    @include setPadding($size-10, $size-10, $size-10, $size-10);
+    @include setFontSize($size-14);
+    @include setPadding($size-6, $size-6, $size-6, $size-6);
```

Comment:
Let's actually leave this as it was before at `$size-10`. Making the tooltip too slim makes it look a little bit out of place in the application.